### ISSUE TITLE
plotPDF fails without  ..,data,... for large numbers of orientations

### ISSUE THREAD

**Issue by kilir** [mtex-toolbox/mtex#81]
Hello Ralf,
I just figured the new syntax of plotPDF. it fails for large numbers of orientations without providing the 'data' argument, or specifying 'points', 'all'.
```
>> plotPDF(ebsd.orientations,hs)
  I'm plotting 833 random orientations out of 1003901 given orientations
  You can specify the the number points by the option "points".
  The option "all" ensures that all data are plotted
Index exceeds matrix dimensions.

Error in orientation/plotPDF (line 70)
  data = data(:,samples,:);
 ```
However `plotPDF(ebsd(1:100).orientations,hs)` or `plotPDF(ebsd.orientations,zeros(length(ebsd),1),hs)` or
`plotPDF(ebsd.orientations,hs,'points','all')` works perfectly.

All the best,
Rüdiger

**Comment by ralfHielscher** [comment]
Hi Rüdiger,

thank you for reporting this. I fixed it.

Ralf.

**Comment by ralfHielscher** [comment]
This should be fixed in MTEX 4.1
